deploy/olm-catalog/kogito-operator/1.0.0-snapshot/kogito-operator.v1.0.0-snapshot.clusterserviceversion.yaml

```diff
@@ -642,7 +647,7 @@ spec:
                     fieldRef:
                       fieldPath: metadata.labels['name']
                 - name: DEBUG
-                  value: "false"
+                  value: "true"
```

Comment:
Please revert that back to `false` since this manifest is used by users to install the operator with OLM. Having DEBUG flag on would fill their logs.